WordPress/Classes/ViewRelated/Post/Revisions/RevisionsTableViewController.swift

```diff
@@ -96,8 +96,11 @@ private extension RevisionsTableViewController {
     }
 
     private func getAuthor(for id: NSNumber?) -> BlogAuthor? {
-        let authors: [BlogAuthor]? = post?.blog.authors?.allObjects as? [BlogAuthor]
-        return authors?.first { $0.userID == id }
+        guard let authorId = id else {
```

Comment:
I realize this isn't a best practice per the style guideline, but due to the double-optional nature of a post and revision.postAuthorId, I struggled to find a cleaner solution. I decided to keep the private function so that configureCell would stay remain succinct.

Any tips for improving this are certainly welcome!

Reply:
If you're referring to the additional guard statement, I think its is fine here 👍

Reply:
Yep, good deal!